app/src/models/avatar.ts

```diff
@@ -13,46 +10,30 @@ export interface IAvatarUser {
   readonly email: string
 
   /** The user's avatar URL. */
-  readonly avatarURL: string
+  readonly avatarURL: string | undefined
```

Comment:
I'm wondering if after these changes we even need the `avatarURL` property... Or even the whole `IAvatarUser` object and instead get the emails on the `<Avatar />` component from from the `CommitIdentity`/`GitAuthor` objects.

From what I see we're only populating `avatarURL` when rendering the avatar on the account settings, so we could probably remove that.

If in the future we need to display avatars for GitHub accounts (e.g when showing PRs), we could have a specific `Avatar` component that just uses the `avatarURL` property that comes from the API.

Anyways, this can/should be done in a separate PR 😃

Reply:
You're right, functionality-wise we don't need it, no. The reason I kept it is that if we've got it then it's a good source of the canonical URL to a user's avatar as opposed to our (albeit well educated) guesses so why not use it?

> Anyways, this can/should be done in a separate PR 😃

:+1: Let's discuss this elsewhere, I was going back and forth on this.